frontend/src/components/PrimaryInfo.vue

```diff
@@ -10,6 +10,20 @@ const state = reactive({
   awayTeamScore: 2,
 });
 
+async function fetchScores() {

```

Comment:
I think this is a good start to getting data to the frontend; however, as we've discussed with Jacob, this will likely need to be a websocket connection for better reactivity, and for the server to push updates instead of having the frontend pull updates. I think we should stick with this for now, especially if it works, but change is as part of #23 or some other ticket(s).